### DISCUSSION TITLE
Whisper degraded performance in Spanish

### DISCUSSION THREAD

**Discussion by adriangalilea** [openai/whisper#1331] [Q&A]
Hello not sure if this is the place for this.

I tried on OpenAI website, finally found the support chatbot and it lead me on loops, spent like an hour trying to reach out... so I try here.

Problem:

Noticed a dramatic degradation on spanish detection, it confuses it for portuguese way too often, it was not the case weeks ago, many people complained about this.

I'm referring to OpenAI whishper via API.

Thank you,

Adrian.

**Comment by jongwook** [comment]
Hello, this site is primarily for the open-source version of Whisper, but the language identification is unfortunately not very accurate (for the models both in this repo and in the API). In case you know the input is in Spanish, you can specify `language=Spanish` in the options to skip the language detection and transcribe Spanish.

**Reply by adriangalilea** [reply]
Regarding your suggestion this is not an option as the input is any  arbitrary language.

My observation however is that:
1 - It detected Spain - spanish correctly 100% of the times. Tested this for weeks with more than a hundred users.
2 - It fails every-time now and all the users are complaining, specially when speaking softly 😆 

I can't simply select spanish as I have some Indian, Filipinos, Japanese and Korean users, plus anyone will speak in english from time to time.

Regarding where to report this, I'm unable to find a better place, if anyone knows where should I go, let me know 👀  support chatbot from OpenAI just keeps me in loops and doesn't allow human support agent at all.

**Comment by notrydo** [comment]
Didn't Whisper integrate with ChatGPT?  Any correlation or am I wrong for
thinking a correlation may exist.

**Reply by adriangalilea** [reply]
What do you mean by "Whisper integrate with ChatGPT" I don't know anything about this

**Comment by notrydo** [comment]
Here is a link to the ChatGPT and Whisper website:
https://openai.com/blog/introducing-chatgpt-and-whisper-apis

I believe it's the same product and so I find it interesting that Whisper
was our open source standard and now our open source model is not working.

Any thoughts?

**Reply by adriangalilea** [reply]
Ah, no no, I'm referring to the API service, I just don't know where to reach out and resorted to here, no idea if the api and the available code here is the same or not.

**Comment by notrydo** [comment]
Is this the same Whisper?  I believe it is.

**Comment by notrydo** [comment]
Got it.
